x-pack/legacy/plugins/task_manager/task_manager.ts

```diff
@@ -259,20 +260,31 @@ export async function claimAvailableTasks(
   logger: Logger
 ) {
   if (availableWorkers > 0) {
-    const { docs, claimedTasks } = await claim({
-      size: availableWorkers,
-      claimOwnershipUntil: intervalFromNow('30s')!,
-    });
+    try {
+      const { docs, claimedTasks } = await claim({
+        size: availableWorkers,
+        claimOwnershipUntil: intervalFromNow('30s')!,
+      });
 
-    if (docs.length !== claimedTasks) {
-      logger.warn(
-        `[Task Ownership error]: (${claimedTasks}) tasks were claimed by Kibana, but (${docs.length}) tasks were fetched`
-      );
+      if (docs.length !== claimedTasks) {
+        logger.warn(
+          `[Task Ownership error]: (${claimedTasks}) tasks were claimed by Kibana, but (${docs.length}) tasks were fetched`
+        );
+      }
+      return docs;
+    } catch (ex) {
+      if (identifyEsError(ex).includes('cannot execute [inline] scripts')) {
+        logger.warn(
```

Comment:
It *will* appear on every poll, but throwing won't prevent that.
My thinking is that throwing will result in polluting the log with both the messaging and the error... feels better to just message what the issue is, no?

As for a global state thing... like you said, it could cause other issues such as ES being sorted but Kibana thinking it can't use TM.